app/models/comment.rb

```diff
@@ -49,6 +49,11 @@ class Comment < ApplicationRecord
   validates :public_reactions_count, presence: true
   validates :reactions_count, presence: true
   validates :user_id, presence: true
+  validates :commentable, on: :create, presence: {
+    message: lambda do |object, _data|
+      "#{object.commentable_type.nil? ? 'item' : object.commentable_type} has been deleted."
```

Comment:
Suggestion: we could use `presence` to make this a bit more compact:

```suggestion
      "#{object.commentable_type.presence || 'item'} has been deleted."
```

Reply:
cool

Reply:
Just wanted to confirm one thing, should I use the commit suggestion button to add the suggested changes or I have to make a new commit with all of the 3 changes that you and rhymes have suggested?

Reply:
You can use the button :)